publication-doi-commons/src/main/java/no/unit/nva/publication/doi/dto/Validatable.java

```diff
@@ -0,0 +1,88 @@
+package no.unit.nva.publication.doi.dto;
+
+import static nva.commons.utils.attempt.Try.attempt;
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.Objects;
+import nva.commons.utils.attempt.Failure;
+import nva.commons.utils.attempt.Try;
+import org.slf4j.Logger;
+
+public abstract class Validatable {
```

Comment:
I find is very odd we should create our own code for supporting `@MandatoryField` annotation, when we have native language constructs that we could use to our benefit instead. 
(optionals)

Reply:
Right now these classes follow not such pattern. I want to make the chain to work with as few changes in the code as possible. Right now the test generators create invalid objects during testing and I want to avoid refactoring them too much